lms/envs/aws.py

```diff
@@ -655,6 +655,8 @@
 # This can be used to separate uploads for different environments
 # within the same S3 bucket.
 ORA2_FILE_PREFIX = ENV_TOKENS.get("ORA2_FILE_PREFIX", ORA2_FILE_PREFIX)
+ORA2_FILEUPLOAD_BACKEND = ENV_TOKENS.get("ORA2_FILEUPLOAD_BACKEND", ORA2_FILEUPLOAD_BACKEND)
+ORA2_FILEUPLOAD_CACHE_NAME = ENV_TOKENS.get("ORA2_FILEUPLOAD_CACHE_NAME", ORA2_FILEUPLOAD_CACHE_NAME)
```

Comment:
Unless we're cherry-picking from upstream (did you check if they've since fixed this?),
we should define these inside the `openedx/stanford` directory..

Reply:
Surprisingly they haven't, should I make an upstream PR or move these into `openedx/stanford`?